test/mocha.setup.js

```diff
@@ -11,26 +11,12 @@ before(function() {
 		.onPost("api/gateway/payment-intents/valid-token-mock/payments").reply(config => {
 			return [ 200, { data: { paymentStatus: 'Paid', requestData: JSON.parse(config.data) } } ];
 		})
+		.onGet("api/gateway/payment-intents/valid-token-mock/stored-cards").reply(config => {
+			return [ 200, { data: [ { maskedCardNumber: '**** **** **** 1234', cardExpiryMonth: 1, cardExpiryYear: 23, encryptedCardData: 'AAAA' } ] } ];
```

Comment:
The masked card number does not follow industry standard masking (0000 00** **** 0000)